### PR TITLE
[Docs] Add inheritance design explanation to Config Tutorial

### PR DESCRIPTION
**Pull request by dreamerlin**

### CODE REVIEW

File: docs/tutorials/1_config.md [hunk 1]
```diff
@@ -1,18 +1,58 @@
...
+The configs that are composed by components from `_base_` are called _primitive_.
+
+For all configs under the same folder, it is recommended to have only **one** _primitive_ config. All other configs should inherit from the _primitive_ config. In this way, the maximum of inheritance level is 3.
+
+For easy understanding, we recommend contributors to inherit from exiting methods.
+For example, if some modification is made base on TSN, users may first inherit the basic TSN structure by specifying `_base_ = ../tsn/tsn_r50_1x1x3_100e_kinetics400_rgb.py`, then modify the necessary fields in the config files.
+
+If you are building an entirely new method that does not share the structure with any of the existing methods, you may create a folder under `configs/TASK`,
+
+If you are building an entirely new method that does not share the structure with any of the existing methods, you may create a folder `xxx_rcnn` under `configs`,
```
Comment (innerlee) [inline docs/tutorials/1_config.md:54]: ？

File: docs/tutorials/1_config.md [hunk 2]
```diff
@@ -398,231 +434,230 @@ which is convenient to conduct various experiments.
 
 ### Config System for Spatio-Temporal Action Detection
 
-We incorporate modular design into our config system,
-which is convenient to conduct various experiments.
+We incorporate modular design into our config system, which is convenient to conduct various experiments.
 
-#### An Example of FastRCNN
+- An Example of FastRCNN
```
Comment (innerlee) [inline docs/tutorials/1_config.md:439]: ？

Comment (dreamerlin) [reply docs/tutorials/1_config.md:439]: Just to keep the Same as above

**Comment by innerlee** [comment]
Why use FastRCNN as example?

**Comment by dreamerlin** [comment]
> ~Why use FastRCNN as example?~
> 
> Okay fine, but FastRCNN is only a part of the model, right?

yeah, spatial temporal detection model is based on FastRCNN but with SlowFast backbone

File: docs/tutorials/1_config.md [hunk 3]
```diff
@@ -398,231 +434,230 @@ which is convenient to conduct various experiments.
...
+        warmup_by_epoch=True,  # Warmup_iters indicates iter num or epoch num
+        warmup_iters=5,   # Number of iters or epochs for warmup
+        warmup_ratio=0.1)   # The initial learning rate is warmup_ratio * lr
+
+    total_epochs = 20  # Total epochs to train the model
+    checkpoint_config = dict(  # Config to set the checkpoint hook, Refer to https://github.com/open-mmlab/mmcv/blob/master/mmcv/runner/hooks/checkpoint.py for implementation
+        interval=1)   # Interval to save checkpoint
+    workflow = [('train', 1)]   # Workflow for runner. [('train', 1)] means there is only one workflow and the workflow named 'train' is executed once
+    evaluation = dict(  # Config of evaluation during training
+        interval=1)  # Interval to perform evaluation
```
Comment (congee524) [inline docs/tutorials/1_config.md:645]: `evaluation = dict(interval=1, save_best='mAP@0.5IOU')`

I think it's better to put `evaluation` on the next line of `data`. Logically, `evaluation` is a dataset setting.

**Review by congee524 — APPROVED** [review]

**Pull request merged**